tests/integration/release/test_runner.py

```diff
@@ -16,13 +16,31 @@
         (["--dataset", "wic", "--year", "2015"]),
         # (["--dataset", "wic", "--population", "USA", "--state", "RI", "--year", "2015"]),
     ],
+    ids=["1", "2", "3", "4"],
```

Comment:
We'd talked at some point about automating these IDs to that (1) they are more descriptive and (2) so that they auto-adjust when you add/remove parameterizations. Maybe: `ids=lambda x: "_".join([item for item in x if not item.startswith("--")]),`